README.md

```diff
@@ -22,13 +22,13 @@ The package is available from CRAN and can be installed by running `install.pack
 To install the latest development version of the package directly from GitHub use the following code:
 
 ```r
-if (!requireNamespace("remotes", quietly = TRUE)) {
+if (!requireNamespace("remotesquietly = TRUE)) {
   install.packages("remotes")
 }
 
-remotes::install_github("pharmaverse/admiral.test", ref = "devel") # This is a required dependency of {admiral}
-remotes::install_github("pharmaverse/admiraldev", ref = "devel") # This is a required dependency of {admiral}
-remotes::install_github("pharmaverse/admiral", ref = "devel")
+remotes::install_github("pharmaverse/admiral.testref = "devel") # This is a required dependency of {admiral}
+remotes::install_github("pharmaverse/admiraldevref = "devel") # This is a required dependency of {admiral}
+remotes::install_github("pharmaverse/admiralref = "devel")
```

Comment:
Something strange happened here with the , and the ref being smooshed into the repo/package name.

Since the default branch is now `devel` we can actually drop the `ref = 'devel'` part of the code